templates/api/knowledge-panels/health/ingredients/ingredient.tt.json

```
@@ -34,9 +34,9 @@
         {
             "element_type": "text",
             "text_element": {
-                "html": `<strong>[% edq(panel.wikipedia_title) %][% sep %]:</strong> [% edq(panel.wikipedia_abstract) %]`,
-                "source_text": `Wikipedia`,
-                "source_url": `[% panel.wikipedia_url %]`,
+                "html": "<strong>[% edq(panel.wikipedia_title) %][% sep %]:</strong> [% edq(panel.wikipedia_abstract) %]",
+                "source_text": "Wikipedia",
+                "source_url": "[% panel.wikipedia_url %]",
```

Perhaps we should add `edq(...)` here too, despite the absence of a call to `lang(...)`.

```suggestion
                "source_url": "[% edq(panel.wikipedia_url) %]",
```